src/main/public/stylesheets/components/_timeline.scss

```diff
@@ -0,0 +1,95 @@
+.timeline-column-1 {
+  display: inline-block;
+  margin-right: 10px;
+  vertical-align: top;
+  width: 28%;
+}
+
+.timeline-column-2 {
+  display: inline-block;
+  width: 67%;
+}
+
+.timeline-column-text {
+  display: inline-block;
+  height: auto;
+  width: 65%;
+}
+
+.timeline-row {
+  display: inline-block;
+  width: 100%;
+
+  .add-event-container {
+    float: left;
+  }
+
+  .calculate {
+    float: right;
+    padding-top: 26px;
+  }
+}
+
+.timeline-rows {
+  div {
+    :not(:first-child) {
+      margin-top: 15px;
+    }
+  }
+}
+
+.timeline-column-4 {
+  display: inline-block;
+  text-align: right;
+  width: 20%;
+}
+
+.timeline-column-3 {
+  display: inline-block;
+  margin-right: 10px;
+  width: 75%;
+}
+
+.add-event-container {
+  margin-top: 15px;
+}
+
+.mobile-show,
+.mobile-table-status {
+  display: none;
+}
+
+@media screen and (max-width: 480px) {
+  .mobile-hide {
```

Comment:
this doesn't look related to timeline, more of a utils sort of thing

Reply:
moved to _mobile.scss